lava_test_plans/testcases/master/template-mmtests.jinja2

```diff
@@ -0,0 +1,22 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set KERNEL_BRANCH = KERNEL_BRANCH|default("") %}
+{% set OS_INFO = OS_INFO|default("") %}
+
+{% set test_name = "mmtests-" + testname %}
+
+{% block test_target %}
+  {{ super() }}
+{% block test_target_prep %}
+{% endblock test_target_prep %}
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/mmtests/mmtests.yaml
+      parameters:
+        SKIP_INSTALL: 'true'
+        TEST_PROG_VERSION: 'a5c1da9c8996'
```

Comment:
Since this will be in preinstalled, I think we have to create a variable for the sha here.
And for us we should set the default to an empty string.
if I read the test-definitions/automated/linux/mmtests/mmtests.sh correctly